### PR TITLE
ticdc: fix pulsar sink documentation error

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-pulsar.md [hunk 1]
```diff
@@ -135,6 +139,32 @@ send-timeout=30
 * 你需要在创建 Changefeed 的时候设置 `protocol` 参数。目前同步数据到 Pulsar 仅支持使用 `canal-json` 协议。
 * `pulsar-producer-cache-size` 参数表示 Pulsar 客户端中缓存 Producer 的数量，因为 Pulsar 的每个 Producer 只能对应一个 topic，TiCDC 采用 LRU 方式缓存 Producer，默认限制为 10240 个。如果你需要同步的 topic 数量大于默认值，则需要调大该数量。
 
+### TLS 加密传输
+
+TiCDC 从 7.5.1 开始支持 Pulsar 的 TLS 加密传输，配置样例如下所示：
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:144]:
```suggestion
TiCDC 从 v7.5.1 和 v8.0.0 开始支持 Pulsar 的 TLS 加密传输，配置样例如下所示：
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 2]
```diff
@@ -170,32 +200,34 @@ send-timeout=30
...
+    # Pulsar mTLS 认证私钥路径
     auth-tls-private-key-path="/data/pulsar/certificate.key"
-    # Pulsar TLS 加密可信证书文件路径
+    # Pulsar mTLS 加密可信证书文件路径
     tls-trust-certs-file-path="/data/pulsar/tls-trust-certs-file"
     ```
 
 - OAuth2 认证
 
+    TiCDC 从 7.5.1 开始支持 Pulsar 的 OAuth2 认证，配置样例如下所示：
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:225]:
```suggestion
    TiCDC 从 v7.5.1 和 v8.0.0 开始支持 Pulsar 的 OAuth2 认证，配置样例如下所示：
```

File: ticdc/ticdc-sink-to-pulsar.md (hunk 1 above)
Comment (qiancai) [reply ticdc/ticdc-sink-to-pulsar.md:144]: 改成这样，避免用户认为 v7.6 也支持

**Review by CharlesCheung96 — APPROVED** [review]

File: ticdc/ticdc-sink-to-pulsar.md [hunk 3]
```diff
@@ -98,12 +98,16 @@ token-from-file="/data/pulsar/token-file.txt"
...
 # Pulsar 使用 basic 账号密码验证身份，此处为密码。
 basic-password="password"
-# Pulsar TLS 加密认证证书路径。
+# Pulsar 使用 mTLS 认证时，客户端的证书路径。
 auth-tls-certificate-path="/data/pulsar/certificate"
-# Pulsar TLS 加密认证私钥路径。
+# Pulsar 使用 mTLS 认证时，客户端的私钥路径。
 auth-tls-private-key-path="/data/pulsar/certificate.key"
-# Pulsar TLS 加密可信证书文件路径。
+# Pulsar TLS 可信证书文件路径，在启用 mTLS 认证或者 TLS 加密时，需要指定该参数。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:105]:
```suggestion
# Pulsar TLS 可信证书文件路径，在 Pulsar 启用 mTLS 认证或者 TLS 加密时，需要指定该参数。
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 4]
```diff
@@ -135,6 +139,32 @@ send-timeout=30
...
+```
+
+config 参数：
+
+```toml
+[sink.pulsar-config]
+tls-trust-certs-file-path="/data/pulsar/tls-trust-certs-file"
+```
+
+如果你的 Pulsar 服务端设置了 `tlsRequireTrustedClientCertOnConnect=true` 参数，那么你需要同时在 changefeed 的配置文件中设置 `tls-key-file-path` 和 `tls-certificate-file` 参数。如下所示：
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:159]: L163 行还提到了 `tls-trust-certs-file-path` 这个参数，也是必须要设置的吗，如果是，可以写到 L159 里

File: ticdc/ticdc-sink-to-pulsar.md [hunk 5]
```diff
@@ -98,12 +98,16 @@ token-from-file="/data/pulsar/token-file.txt"
...
 auth-tls-certificate-path="/data/pulsar/certificate"
-# Pulsar TLS 加密认证私钥路径。
+# Pulsar 使用 mTLS 认证时，客户端的私钥路径。
 auth-tls-private-key-path="/data/pulsar/certificate.key"
-# Pulsar TLS 加密可信证书文件路径。
+# Pulsar TLS 可信证书文件路径，在启用 mTLS 认证或者 TLS 加密时，需要指定该参数。
 tls-trust-certs-file-path="/data/pulsar/tls-trust-certs-file"
+# Pulsar 启用 TLS 加密时，加密私钥路径。
+tls-key-file-path="/data/pulsar/tls-key-file"
+# Pulsar 启用 TLS 加密时， 加密证书文件路径。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:109]:
```suggestion
# Pulsar 启用 TLS 加密时，客户端的加密私钥路径。
tls-key-file-path="/data/pulsar/tls-key-file"
# Pulsar 启用 TLS 加密时， 客户端的加密证书文件路径。
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 6]
```diff
@@ -98,12 +98,16 @@ token-from-file="/data/pulsar/token-file.txt"
 basic-user-name="root"
 # Pulsar 使用 basic 账号密码验证身份，此处为密码。
 basic-password="password"
-# Pulsar TLS 加密认证证书路径。
+# Pulsar 使用 mTLS 认证时，客户端的证书路径。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:101]:
```suggestion
# Pulsar 启用 mTLS 认证时，客户端的证书路径。
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 7]
```diff
@@ -98,12 +98,16 @@ token-from-file="/data/pulsar/token-file.txt"
...
 auth-tls-certificate-path="/data/pulsar/certificate"
-# Pulsar TLS 加密认证私钥路径。
+# Pulsar 使用 mTLS 认证时，客户端的私钥路径。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:103]:
```suggestion
# Pulsar 启用 mTLS 认证时，客户端的私钥路径。
```

File: ticdc/ticdc-sink-to-pulsar.md (hunk 6 above)
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:101]:
```suggestion
# Pulsar 启用 mTLS 认证时，客户端的证书路径。
```

File: ticdc/ticdc-sink-to-pulsar.md (hunk 7 above)
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:103]:
```suggestion
# Pulsar 启用 mTLS 认证时，客户端的私钥路径。
```

File: ticdc/ticdc-sink-to-pulsar.md [hunk 8]
```diff
@@ -170,32 +200,34 @@ send-timeout=30
...
     ```toml
     [sink.pulsar-config]
-    # Pulsar TLS 加密认证证书路径
+    # Pulsar mTLS 认证证书路径
     auth-tls-certificate-path="/data/pulsar/certificate"
-    # Pulsar TLS 加密认证私钥路径
+    # Pulsar mTLS 认证私钥路径
     auth-tls-private-key-path="/data/pulsar/certificate.key"
-    # Pulsar TLS 加密可信证书文件路径
+    # Pulsar mTLS 加密可信证书文件路径
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-pulsar.md:219]:
```suggestion
    # Pulsar mTLS 可信证书文件路径
```

Comment (qiancai) [reply ticdc/ticdc-sink-to-pulsar.md:219]: 和 L105 一致，@asddongmen PTAL

**Review by lilin90 — COMMENTED** [review]
/lgtm

**Review by qiancai — APPROVED** [review]

**Pull request merged**